sdk/communication/AzureCommunicationChat/Source/Signaling/Events/TrouterEvent.swift

```diff
@@ -65,7 +65,9 @@ public enum TrouterEvent {
         }
 
         switch chatEventId {
-        case ChatEventId.chatMessageReceived:
+        case .realTimeNotificationConnected, .realTimeNotificationDisconnected:
+            throw AzureError.client("Chat Event ID should not be \(chatEventId)")
+        case .chatMessageReceived:
```

Comment:
Again, AzureError here implies that the customer can do something about this, which they can't. If this code is reached, it could only be due to a bug in the above code since it should exit by this point.
My recommendation would be to just do a logger.error here and return the relevant event. Because that's what you would do if lines 54-65 were deleted (which you totally could do if you moved the guard statement.

Reply:
It seems not worth adding the logger to the file. I changed to ignore the bug cases.